handlers/auth/authorize.js

```diff
@@ -7,7 +7,7 @@ module.exports = async function authorize(req, res) {
     const authorizationUrl = client.authorizationUrl({
       redirect_uri: conf.auth.redirect_uri,
       response_type: 'code',
-      scope: 'openid',
+      scope: conf.auth.scope,
```

Comment:
With this adjustment the change stays backwards-compatible (i.e. users with existing configuration files do not need to change them).

```suggestion
      scope: conf.auth.scope ?? 'openid',
```

Reply:
Thanks, great suggestion!